src/main/java/org/nmfw/foodietree/domain/store/repository/LoginIdCheckMapper.java

```diff
@@ -0,0 +1,19 @@
+package org.nmfw.foodietree.domain.store.repository;
+
+import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Param;
+
+@Mapper
+public interface LoginIdCheckMapper {
+
+    // 중복 확인 (아이디 = 이메일)
+    /**
+     * @param type - 어떤걸 중복검사할지 (ex: account = email)
+     * @param keyword - 중복검사할 실제값
+     * @return - 중복이면 true, 아니면 false
+     */
+    boolean existsById(
+            @Param("type") String type,
```

Comment:
type 이 account 인 경우에 대해서 중복처리하는 메소드라서 existsById로 이름 지은건 알겠습니다. 그런데 매개변수 type 을 사용하는 의미가 없어보입니다.
나중을 고려해서 메소드 이름을 existsByType 으로 명명한다면 type에 따라서 중복을 확인하는 메소드로 역할이 확실해질거 같아요

```suggestion
    boolean existsByType(
            @Param("type") String type,
```